File: 30 Day Coding Challenge/29 September 2023.py
#  Question 1

import sys

def duplicateNumber(arr, n) :
    #Your code goes here
    for i in range(n):
        for j in range(n):
            if arr[i]==arr[j] and i!=j:
                return arr[i]

    return 0;

# Question 2


import sys

def intersections(arr1, n, arr2, m) :
    #Your code goes here
    for i in range(n):
        for j in range(m):
            if arr1[i]==arr2[j]:
                print(arr1[i], end = " ")
                arr2[j]= sys.maxsize
                break

File: 30 Day Coding Challenge/29 September 2023.java
// Question 1
public class Solution{  
    
    public static int duplicateNumber(int arr[]) {
    	//Your code goes here
        for(int i =0;i<arr.length; i++){
            for(int j=0; j<arr.length; j++){
                if(arr[i] == arr[j] && i!=j){
                    return arr[i];
                }
            }
        }
        return 0;
    }
}


